internal/schema-registry/command_kek_update.go

```diff
@@ -0,0 +1,84 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	srsdk "github.com/confluentinc/schema-registry-sdk-go"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+)
+
+func (c *command) newKekUpdateCommand(cfg *config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update <name>",
+		Short: "Update a KEK.",
```

Comment:
What does KEK stand for?

Reply:
kek: key encryption key. dek: data encryption key.

Reply:
Can we include that ("Key Encryption Key (KEK)") in the user-facing descriptions?